devpack/src/main/java/io/neow3j/devpack/compiler/JVMOpcode.java

```diff
@@ -0,0 +1,263 @@
+package io.neow3j.devpack.compiler;
+
+// Taken from com.sun.tools.classfile.Opcode
+public enum JVMOpcode {
```

Comment:
WOW! Nice.

Is it compatible to JDK8? Or which Java specification? It would be nice to put on the header as a comment.